scripts/api_master.py

```diff
@@ -1964,6 +1964,14 @@
                         "keras.utils.Config",
                     ],
                 },
+                {
+                    "path": "resource_management_utils",
+                    "title": "Resource management utilities",
```

Comment:
I would prefer for the title to be Rematerialization.
@hertschuh do you have any thoughts?

Reply:
Yes, let's call it "Rematerialization"